shell/src/test/java/alluxio/cli/fsadmin/command/SummaryCommandTest.java

```diff
@@ -0,0 +1,193 @@
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.cli.fsadmin.report.SummaryCommand;
+import alluxio.client.MetaMasterClient;
+import alluxio.client.block.BlockMasterClient;
+import alluxio.Configuration;
+import alluxio.ProjectConstants;
+import alluxio.PropertyKey;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.util.network.NetworkAddressUtils;
+import alluxio.util.network.NetworkAddressUtils.ServiceType;
+import alluxio.wire.BlockMasterInfo;
+import alluxio.wire.BlockMasterInfo.BlockMasterInfoField;
+import alluxio.wire.MasterInfo;
+import alluxio.wire.MasterInfo.MasterInfoField;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class SummaryCommandTest {
+  private MetaMasterClient mMetaMasterClient;
+  private String mAddress;
+  private int mWebPort;
+  private int mRpcPort;
+  private long mStartTimeMs;
+  private long mUpTimeMs;
+  private String mVersion;
+  private boolean mSafeMode;
+
+  private BlockMasterClient mBlockMasterClient;
+  private int mLiveWorkerNum;
+  private int mLostWorkerNum;
+  private long mCapacityBytes;
+  private Map<String, Long> mCapacityBytesOnTiers;
+  private long mUsedBytes;
+  private Map<String, Long> mUsedBytesOnTiers;
+  private long mFreeBytes;
+
+  private File mFile;
+  private PrintStream mPrintStream;
+
+  @Before
+  public void prepareDependencies() throws IOException {
+    setInfoValues();
+
+    // Prepare mock meta master client
+    mMetaMasterClient = Mockito.mock(MetaMasterClient.class);
+    MasterInfo masterInfo = new MasterInfo().setMasterAddress(mAddress)
+        .setWebPort(mWebPort).setRpcPort(mRpcPort).setStartTimeMs(mStartTimeMs)
+        .setUpTimeMs(mUpTimeMs).setVersion(mVersion).setSafeMode(mSafeMode);
+    Set<MasterInfoField> masterInfoFilter = new HashSet<>(Arrays
+        .asList(MasterInfoField.MASTER_ADDRESS, MasterInfoField.WEB_PORT,
+            MasterInfoField.RPC_PORT, MasterInfoField.START_TIME_MS,
+            MasterInfoField.UP_TIME_MS, MasterInfoField.VERSION,
+            MasterInfoField.SAFE_MODE));
+    Mockito.when(mMetaMasterClient.getMasterInfo(masterInfoFilter)).thenReturn(masterInfo);
```

Comment:
you could replace `masterInfoFilter` with `Mockito.any()` so that you don't need to specify all the fields here.

Reply:
Good and easy to implement way! Thanks